### PR TITLE
Ui update

### PR DESCRIPTION
**Pull request by cburnham4**
Ui update pr by @Angelito13

### CODE REVIEW

**Comment by Angelito13** [comment]
The development team changed. Not sure if that matters but just curious about it

File: LunarTimes/TabBarViewController.swift [hunk 1]
```diff
@@ -137,3 +149,16 @@ extension TabBarViewController: CLLocationManagerDelegate {
         })
     }
 }
+
+extension UIImage {
+
+   class func imageWithColor(color: UIColor, size: CGSize) -> UIImage {
```
Comment (cburnham4) [inline LunarTimes/TabBarViewController.swift]: format this: Cmd + A -> ctrl + I

File: LunarTimes/GradientView.swift [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+    }
+    
+    @IBInspectable var secondColor: UIColor = UIColor.clear {
+        didSet {
+            updateView()
+        }
+    }
+    @IBInspectable var dropShadow: Bool = false {
+        didSet {
+            updateView()
```
Comment (cburnham4) [inline LunarTimes/GradientView.swift]: each of these should not reset the entire view when updated.

File: LunarTimes/GradientView.swift [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+    }
+    
+    func updateView() {
+        let layer = self.layer as! CAGradientLayer
+        layer.startPoint = CGPoint(x: 0.0, y: 0.0)
+        layer.endPoint = CGPoint(x: 1.0, y: 1.0)
+        layer.colors = [ firstColor.cgColor, secondColor.cgColor]
+        layer.locations = [NSNumber(value: 0.5), NSNumber(value: 0.35)]
+        
+        if dropShadow {//Make a drop shadow
```
Comment (cburnham4) [inline LunarTimes/GradientView.swift]: Make this a seperate func

**Pull request merged**